apps/notifications/package-lock.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
apps no longer have their own package locks. You should be able to drop this file entirely. After you add the dependency to notifications/package.json, run `npm i` in the root until you get a stable package-lock.json in the root.

Reply:
thanks @blowery - I updated but it didn't seem to address the the issue at hand, not sure if it was supposed to.